src/ElectricPokemon.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private String type = "electric";
```

Comment:
deze mag zelfs `private static final String` zijn en de naam mag in hoofdletters `TYPE`, want dat is de conventie wanneer je constanten gebruikt in je klasse